modules/webauthorprofile/lib/webauthorprofile_corefunctions.py

```diff
@@ -430,7 +431,10 @@ def get_orcid_pubs(person_id):
 
     external_pubs = dict()
     external_pubs['arxiv'] = get_arxiv_pubs(person_id)
-    external_pubs['doi'] = get_orcid_pubs(person_id)
+    try:
+        external_pubs['doi'] = get_orcid_pubs(person_id)
+    except:
```

Comment:
Mmh. Don't you have to always have defined `external_pubs['doi']`? At least to empty list or something?